core/storage/base_model/gae_models_test.py

```diff
@@ -43,6 +45,10 @@ def test_error_cases_for_get_method(self):
         self.assertIsNone(
             base_models.BaseModel.get('Invalid id', strict=False))
 
+    def test_export_data_raises_not_implemented_error(self):
```

Comment:
Rename this test as well, an error is raised *because* it's the base model, it's not a universal fact about `export_data`. That should be clear from the test name, but it currently isn't.

Reply:
Done